apps/profiles/viewsets.py

```diff
@@ -116,6 +120,10 @@ class FollowerViewSet(viewsets.ModelViewSet):
     permission_classes = (IsAuthenticated, FollowerPermissions)
     http_method_names = ["get", "post", "delete"]
     lookup_url_kwarg = "user_id"
+    filter_backends = [TrigramSimilaritySearchFilter, OrderingFilter, DjangoFilterBackend]
+    filterset_class = UserFilter
```

Comment:
filterset class was needed in events only to handle annotated fields like average_rating, the simplest way to add fields would be just defining filterset_fields like:
```
    filterset_fields = {
        'name': ['exact', 'icontains'],
        'start_date': ['exact', 'gte', 'lte'],
        'end_date': ['exact', 'gte', 'lte'],
        'rating': ['exact', 'gte', 'lte'],
        'tags__name': ['exact', 'in'],
        'category__name': ['exact', 'in'],
        'city': ['exact', 'in'],
        'free': ['exact'],
        'participants_age': ['exact', 'gte', 'lte'],
    }
```

Reply:
this way we don't need custom filter